tests/Support/DbHelper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Log\Target\Db\Tests\Support;
+
+use Yiisoft\Db\Driver\Pdo\PdoConnectionInterface;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+
+use function explode;
+use function file_get_contents;
+use function trim;
+
+final class DbHelper
+{
+    /**
+     * Loads the fixture into the database.
+     *
+     * @throws Exception
+     * @throws InvalidConfigException
+     */
+    public static function loadFixture(PdoConnectionInterface $db, string ...$fixtures): void
+    {
+        $db->open();
+
+        foreach ($fixtures as $fixture) {
```

Comment:
How about using query builder instead of executing SQL from files? This way we can set table prefixes same was as it was possible with migrations.